src/sentry/api/endpoints/organization_details.py

```diff
@@ -261,6 +262,20 @@ def validate_projectRateLimit(self, value):
             )
         return value
 
+    def validate_providerName(self, value):
+        from sentry.auth import manager
+
+        has_api_auth_provider = features.has(
+            "organizations:api-auth-provider",
+            self.context["organization"],
+            actor=self.context["request"].user,
```

Comment:
I don't think you need `actor` here. The feature flag doesn't/shouldn't change for organization even if a superuser s looking at it.

Reply:
Nit: The code style that you're declaring this is different from above, see Line 232.

We want to try to keep the coding style consistent.